conf/defaults.ini

```diff
@@ -812,6 +812,7 @@ use_refresh_token = false
 #################################### Basic Auth ##########################
 [auth.basic]
 enabled = true
+password_policy = false
```

Comment:
Please add a short description of the setting.

Reply:
Updated as requested